test/common/upstream/health_check_fuzz.proto

```diff
@@ -80,6 +80,7 @@ message Action {
     //TODO: respondBody, respondTrailers
     google.protobuf.Empty trigger_timeout_timer = 3;
     RaiseEvent raise_event = 4 [(validate.rules).enum.defined_only = true];
+    bool raise_go_away = 5;
```

Comment:
Reading this, on may think that the boolean flag states whether go-away should be raised or not.
It should either be changed to an enum (NoError vs Other/Error), or at least add a comment clarifying this.

Reply:
Good point. Let me change.

Reply:
Will change to enum.